services/scabbard/cli/man/scabbard-ns-create.1.md

```diff
@@ -0,0 +1,94 @@
+% SCABBARD-NS-CREATE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**scabbard-ns-create** — Creates a Sabre namespace.
+
+SYNOPSIS
+========
+
+**scabbard ns create** \[**FLAGS**\] \[**OPTIONS**\] NAMESPACE
+
+DESCRIPTION
+===========
+This command allows users to create a new Sabre namespace in state for the
```

Comment:
Could use brief information on what this Sabre namespace is used for. (Or why someone would use this command.) Does this define the area of state that the node's scabbard service will use for storing {everything it needs to store}?

Reply:
Added a brief description of namespaces. @agunde406 can you validate that the description is accurate?

Reply:
lgtm